src/platform/Tizen/DnssdImpl.cpp

```diff
@@ -41,7 +41,8 @@ using namespace chip::DeviceLayer::Internal;
 
 namespace {
 
-constexpr uint8_t kDnssdKeyMaxSize = 32;
+constexpr uint8_t kDnssdKeyMaxSize       = 32;
+constexpr const char * kEmptyAddressIpv6 = "0000:0000:0000:0000:0000:0000:0000:0000";
```

Comment:
is this the only way IP addresses are formatted (fully expanded)? I would expect "::" to be the same as 0 as well.

Reply:
Maybe for safety (if someday Tizen will change string representation) we could use the output from `chip::Inet::IPAddress::FromString(ipv6, ipAddr)` to check whether the address is correct? In case of all zeros, the `ipAddr` type should equal to `IPAddressType::kAny`.  The `chip::Inet::IPAddress::FromString` is used here anyway.